lib/sensu/settings/validators/client.rb

```diff
@@ -110,6 +110,8 @@ def validate_client(client)
               invalid(client, "client name cannot contain spaces or special characters")
             must_be_a_string(client[:address]) ||
               invalid(client, "client address must be a string")
+            must_be_a_string_if_set(client[:client_key]) ||
```

Comment:
shouldn't this attribute just be `key`? Following the normal definition attribute naming convention? Using scope.